Gopkg.toml

```diff
@@ -44,7 +44,7 @@
 
 [[constraint]]
   name = "github.com/deislabs/oras"
-  version = "~0.3.3"
+  revision = "12b336a3ea9d5d048f6726adbfc0a8271e75e2a8"
```

Comment:
is there plans to release a v0.4.0? If so should we wait for that to be released before merging?

Reply:
We can certainly hold off, should be a few days.

Reply:
done